### ISSUE TITLE
[v7.1.3 Beta]  npm report: 45 vulnerabilities (20 moderate, 23 high, 2 critical)

### ISSUE THREAD

**Issue by Georg-Git** [SVG-Edit/svgedit#762]
I liked this SVG Editor - so - **many thanks for reviving this project!!**

I succeeded to build a fresh distribution from git master (currently 7.1.3 Beta) via npm install and build.

But the npm install process reported:
45 vulnerabilities (20 moderate, **23 high, 2 critical**)

Puh - that is still a long way to go ...

**Comment by Georg-Git** [comment]
OK - the npm report makes me nervous.
But I found the reported vulnerabilities are only related to development environment, especially the libraries of the webserver.

So I had removed some stuff for the development environment and focus on building the SVG-Editor (HTML, JS Files etc.)

Now NPM audit reported:
"found 0 vulnerabilities"

Still some warnings about deprecated  libraries, but most of the libraries are only for development purposes.

But one warning catched my eyes:
"core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues."
(https://www.npmjs.com/package/core-js/v/2.6.12)

A lot of included libraries in SVG-Edit depend on core-js, but only @babel/polyfill fixed the core-js version to the old "2.6.12".
I had removed the dependency of @babel/polyfill (only needed for older Browsers?) and set the core-js version to the latest.

Rebuilding SVG-Edit - and it seems to work with my Chrome Browser without any Problems.

So - what is this now?
It is not a bug report.
It is also not really a reporting of a known security issue.
A kind of feature request to replace @babel/polyfill due to the dependency on a not recommended old version of core-js.
😊
